### ISSUE TITLE
Add inline mock command

### ISSUE THREAD

**Issue by DLMousey** [DLMousey/sarcasm-bot#17]
Add a command to sarcasm bot that allows for inline mocking;
```
!mock text goes here
// delete OP's message
// replace with sarcasm case version
```

I should've added this last release but caught a case of tunnel vision mocking a specific message instead of the last one a user sent

**Comment by liamjcooper** [comment]
@DLMousey in the case of an inline mock would you still want the bots response to mention the original author as the first part of the message or just have the text as is?
<img width="224" alt="Discord_2022-02-25_23-07-15" src="https://user-images.githubusercontent.com/17281410/155814919-f5a8a892-42d9-4d6b-9fb3-ccde0f696d59.png">

**Comment by DLMousey** [comment]
Not sure actually - i think it'd be best to leave it off. `!mock` is targeted at a specific user so you want them to know you're taking the mickey but i'm imagining this particular functionality would be used in conversation to quote someone (off discord) saying a dumb

**Comment by DLMousey** [comment]
Released in [v0.4.0](https://github.com/DLMousey/sarcasm-bot/releases/tag/0.4.0), closing this issue
